universal-application-tool-0.0.1/test/forms/QuestionFormTest.java

```diff
@@ -49,29 +49,4 @@ public void getBuilder_returnsCompleteBuilder() throws Exception {
 
     assertThat(actual).isEqualTo(expected);
   }
-
```

Comment:
suggestion: can we keep these test cases since this logic still exists?

Reply:
we could, but the main method for making paths is `QuestionService#getPath` and the temporary path method is going to be deleted in a follow-up PR.

Reply:
That should say `ReadOnlyQuestionService#makePath`, right?

Reply:
yup! edited the message correct it!